cluster/manifests/appdynamics/secrets/appdynamics-sandbox-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: appdynamics-prod
+type: Opaque
+data:
+  access_key: "{{ .ConfigItems.appd_prod_acesskey | base64 }}" 
+  account_name: "{{ .ConfigItems.appd_prod_account | base64 }}"
+  event_service_key: "{{ .ConfigItems.appd_prod_es_key | base64 }}"
+  global_account_name: "{{ .ConfigItems.appd_prod_global_account | base64 }}"
+  uri: "{{ .ConfigItems.appd_prod_uri | base64 }}"
+  es_port: "{{ .ConfigItems.appd_prod_es_port | base64 }}"
+  es_admin_port: "{{ .ConfigItems.appd_prod_es_admin_port | base64 }}"
+  es_uri: "{{ .ConfigItems.appd_prod_es_uri | base64 }}" 
```

Comment:
This looks identical to the secret above (`appdynamics-prod-secrets.yaml`), lets reduce it to only one, the content can still be different depending on environment.